nicintel_spi.c

```diff
@@ -228,6 +217,65 @@ int nicintel_spi_init(void)
 	if (register_shutdown(nicintel_spi_shutdown, NULL))
 		return 1;
 
+	return 0;
+}
+
+static int nicintel_spi_i210_enable_flash()
+{
+	uint32_t tmp;
+
+	tmp = pci_mmio_readl(nicintel_spibar + FLA);
+	if (tmp & (1 << FL_LOCKED)) {
```

Comment:
I was trying to use the conventions of the driver. The BIT macro is not used in the whole file. If you want I can add a patch to convert the whole driver from << to BIT.